src/test/java/oilinjection/util/ParserTest.java

```diff
@@ -0,0 +1,63 @@
+package oilinjection.util;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import java.util.Arrays;
+import java.util.List;
+import oilinjection.domain.vo.RentInfo;
+import org.assertj.core.api.Assertions;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.EmptySource;
+import org.junit.jupiter.params.provider.ValueSource;
+
+class ParserTest {
+
+    @DisplayName("올바른 입력이 들어오면, 정상적으로 실행된다.")
+    @Test
+    void 정상_입력1() {
+        final String input = "car1:200";
+        final RentInfo rent = new RentInfo("car1", 200);
+        assertThat(Parser.parse(input)).isEqualTo(Arrays.asList(rent));
+    }
+
+    @DisplayName("두 개 이상의 rent 정보가 올바른 입력 형식으로 들어와도 정상적으로 실행된다.")
+    @Test
+    void 정상_입력2() {
+        final String input = "car1:200,car2:300";
+        final List<RentInfo> rents = Arrays.asList(new RentInfo("car1", 200), new RentInfo("car2", 300));
+        assertThat(Parser.parse(input)).isEqualTo(rents);
```

Comment:
`Parser.parse()`가 순서도 보장하는 List를 반환할거라는 확신이 없을 것 같아요! 현재 테스트는 `car2`객체가 먼저오면 실패를 할 수 있는 테스트인 것 같아요! 그래서 저는 보통 이런 테스트는 `assertAll`을 활용해서 아래와 같이 작성합니다!

```suggestion
        List<RentInfo> actual = Parser.parse(input);
        assertAll(
            () -> assertThat(actual).hasSize(2), 
            () -> assertThat(actual).containsAll(rents)
        );
```

Reply:
오.. 리뷰어님 덕분에 `assertAll` 에 대해 알게 되었습니다! 😄
선호 리뷰어님 말씀대로 순서가 바뀐다면, 해당 테스트는 실패하게 되네요! `size`와 `포함하는지에 대한 여부`를 판단하는 형식으로 변경하겠습니다!